spec/verb_list_comparisor_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'verb_list_comparisor'
+
+describe VerbListComparisor do
+  let(:csv_path) { 'test-verb-list.csv' }
+  subject(:verb_list_comparisor) { VerbListComparisor.new(csv_path) }
+
+  describe '#compare_databases' do
+    context 'when logged in' do
```

Comment:
there is no setup for this context, so it's functionally meaningless... do you need to assert a distinction between logged in or not for this VerbListComparisor functionality? if you're unit testing, authentication would be an unrelated concern I believe.